src/main/java/org/jenkinsci/plugins/DependencyCheck/aggregator/FindingsAggregator.java

```diff
@@ -48,7 +51,7 @@ public SeverityDistribution getSeverityDistribution() {
     }
 
     public List<Finding> getAggregatedFindings() {
-        return aggregatedFindings;
+        return aggregatedFindings.stream().collect(Collectors.toList());
```

Comment:
is the goal obtain a unmodifiable list? or you want provide a copy of the internal list?

Reply:
Because aggregatedFindings is now a SortedSet that does not implement List interface, I'm converting it to List using this one-liner to avoid making excessive changes to the rest of the aggregator.

Reply:
Has commented above is not needed change the type of aggregatedFindings